sourcegraph/sourcegraph
# admin-analytics: add "insights" page

Closes https://github.com/sourcegraph/sourcegraph/issues/41248.

## Test plan
- `sg start enterprise-codeinsights`
- Open http://localhost:3080/site-admin/analytics/code-insights
- Check the page and metrics are showing correctly

## Screenshots
| Description | Screenshot |
| --: | :-- |
| Analytics / Insights | <img width="1392" alt="image" src="https://user-images.githubusercontent.com/6717049/189542368-a0c49661-82e4-4544-af10-6634a6eb72bd.png"> |
| Analytics / Overview | <img width="1392" alt="image" src="https://user-images.githubusercontent.com/6717049/189542404-d36d631d-beba-4c9e-9196-b9d72f0dc83d.png"> |
